apps/wallet-dashboard/components/SupplyIncreaseVestingOverview.tsx

```diff
@@ -0,0 +1,153 @@
+                    queryKey: ['get-timelocked-staked-objects', account?.address],
+                });
+                queryClient.invalidateQueries({
+                    queryKey: [
+                        'get-all-owned-objects',
+                        account?.address,
+                        {
+                            StructType: TIMELOCK_IOTA_TYPE,
+                        },
+                    ],
+                });
+            });
+    }
+
+    return showSupplyIncreaseVestingOverview ? (
+        <div style={{ gridArea: 'vesting' }} className="with-vesting flex grow overflow-hidden">
+            <Panel>
+                <Title title="Vesting" />
+                <div className="flex h-full w-full items-center gap-md p-md--rs">
+                    <div className="w-1/2">
+                        <Card type={CardType.Filled}>
+                            <CardBody
+                                title=""
+                                subtitle={
+                                    <LabelText
+                                        size={LabelTextSize.Large}
+                                        label="Next reward"
+                                        text={
+                                            nextPayoutResult.isPending
+                                                ? '-'
+                                                : `${formattedNextPayout}   `
+                                        }
+                                        supportingLabel={nextPayoutSymbol}
+                                    />
+                                }
+                            />
+                            <CardAction
+                                type={CardActionType.Button}
+                                buttonType={ButtonType.Ghost}
+                                title={formattedLastPayoutExpirationTime}
+                                icon={<SvgClock />}
+                            />
+                        </Card>
+                    </div>
+                    <div className="w-1/2">
+                        <Card type={CardType.Filled}>
+                            <CardBody
+                                title=""
+                                subtitle={
+                                    <LabelText
+                                        size={LabelTextSize.Large}
+                                        label="Available for staking"
+                                        text={formattedAvailableStaking}
+                                        supportingLabel={availableStakingSymbol}
+                                    />
+                                }
+                            />
+                            <CardAction
+                                type={CardActionType.Button}
+                                buttonType={ButtonType.Primary}
+                                title={'Stake'}
+                                onClick={() => handleNewStake()}
```

Comment:
just a comment that we could disable this button if there is not enough to stake, but we can do this in a different PR, i will track it

Reply:
It's a small thing, I will do it here